Atomic/syscontainers.py

```diff
@@ -1354,6 +1353,8 @@ def prune_ostree_images(self):
         refs = {}
         app_refs = []
 
+        self.prune_storage(repo)
```

Comment:
prune must be done explicitly with "atomic images prune".  I changed the function to be not public though